+++ android2.js 2015-12-07 09:10:26.416080722 +0000
@@ -294,6 +294,7 @@
     _.each(this.args.chromeOptions, function (val, option) {
       if (typeof caps.chromeOptions[option] === "undefined") {
         caps.chromeOptions[option] = val;
+        caps.xwalkOptions[option] = val;
       } else {
         logger.warn("Cannot pass option " + caps.chromeOptions[option] + " because Appium needs it to make chromeDriver work");
       }